package.json

```diff
@@ -12,9 +12,8 @@
   },
   "scripts": {
     "build": "lage build --verbose",
-    "build:codesandbox": "yarn build --to @fluentui/react --to @fluentui/react-components --min",
+    "build:codesandbox": "yarn build --to @fluentui/react --to @fluentui/react-components",
     "build:fluentui:docs": "gulp build:docs",
-    "build:min": "yarn build --to @fluentui/react --to @fluentui/react-northstar --to @fluentui/react-components --min",
```

Comment:
what is build:min ???

Reply:
no idea why this exist tbh. not used anywhere so I removed it.